charts/rancher-gatekeeper-operator/0.1.1/Chart.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+description: A Helm chart for Gatekeeper
+name: rancher-gatekeeper-operator
+keywords:
+  - open policy agent
+version: 0.1.0
+home: https://github.com/open-policy-agent/gatekeeper
+sources:
+  - https://github.com/open-policy-agent/gatekeeper.git
+appVersion: v3.1.0-beta.7
```

Comment:
This should also be updated to the new image to match.

Reply:
done